crates/cli/src/utils/cmd.rs

```diff
@@ -447,14 +449,21 @@ pub async fn print_traces(
     result: &mut TraceResult,
     decoder: &CallTraceDecoder,
     verbose: bool,
+    json: bool,
+    state_changes: bool,
 ) -> Result<()> {
     let traces = result.traces.as_mut().expect("No traces found");
 
     println!("Traces:");
     for (_, arena) in traces {
```

Comment:
if `json` is active this `println!` should not render

This will allow directly passing to `jq` for parsing

Reply:
Updated, thanks! Also updated [here](https://github.com/foundry-rs/foundry/pull/9013/files#diff-448a805ef6f60ed32e529c390289b7064010d85b3da89d32d3a6bd4766723769R182) when `-q` option is used.